elements/pfe-button/src/pfe-button.json

```
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Button",
+  "description": "",
+  "type": "object",
+  "tag": "pfe-button",
+  "class": "PfeButton",
+  "category": "content",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object",
+      "properties": {
+        "default": {
+          "title": "Default slot",
+          "type": "array",
+          "namedSlot": false,
+          "items": {
+            "oneOf": [
+              {
+                "$ref": "raw"
```

```suggestion
                "$ref": "button"
```

Might be good to be specific here since the component only accepts the button tag